src/freenas/usr/local/lib/middlewared_truenas/alert/source/failover_sync.py

```diff
@@ -32,3 +32,17 @@ class FailoverKeysSyncFailedAlertClass(AlertClass, SimpleOneShotAlertClass):
     )
 
     products = ("ENTERPRISE",)
+
+
+class FailoverKMIPKeysSyncFailedAlertClass(AlertClass, SimpleOneShotAlertClass):
+    deleted_automatically = False
+
+    category = AlertCategory.HARDWARE
```

Comment:
We have `HA` alert category, please move it there. `HARDWARE` is for hardware faults, kayako tickets will be created for those which is very undesirable.